src/Bindings/Plugin.h

```diff
@@ -74,6 +74,8 @@ class cPlugin
 	virtual bool OnPlayerBrokenBlock        (cPlayer & a_Player, int a_BlockX, int a_BlockY, int a_BlockZ, char a_BlockFace, BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta) = 0;
 	virtual bool OnPlayerDestroyed          (cPlayer & a_Player) = 0;
 	virtual bool OnPlayerEating             (cPlayer & a_Player) = 0;
+	virtual bool OnPlayerEditedBook         (cPlayer & a_Player, const cBookContent & a_BookContent, bool a_IsSigned) = 0;
+	virtual bool OnPlayerEditingBook        (cPlayer & a_Player, const cBookContent & a_BookContent, bool a_IsSigned) = 0;
```

Comment:
That works with the current code. I tested it and I can get the current content of the book in in the hooks and can modify it. It work's fine. Do you want test it? A example plugin?